### ISSUE TITLE
Register

### ISSUE THREAD

**Issue by stephlocke** [lockedata/rcms#43]
## Detailed task
Buy a ticket for the conference

## Assessing the task
Try to perform the task. Use google and the system documentation to help - part of what we're trying to assess how easy it is for people to work out how to do tasks.

Use a 👍 (`:+1:`) reaction to this task if you were able to perform the task. Use a 👎 (`:-1:`) reaction to the task if you could not complete it. Add a reply with any comments or feedback.

## Extra Info
- Site: [odoo](//http://188.166.159.192:8069)
- System documentation: [odoo docs](https://www.odoo.com/page/docs)
- Role: Attendee
- Area: Processes

**Comment by Amymcdougall** [comment]
## Extra Info

Site: [odoo](http://188.166.159.192:8069)

**Comment by Amymcdougall** [comment]
Torben to work on these issues, ready for Naras to check later. @tvede @naras

**Comment by tvedebrink** [comment]
:-1: I can't login as I don't know how to sign up a new account

**Comment by Amymcdougall** [comment]
I can buy tickets for a event- under multiple email addresses.

**Comment by stephlocke** [comment]
@Amymcdougall can you provide your steps to purchase so @tvedebrink can give it another go?

**Comment by tvedebrink** [comment]
For fun I tried to open the "Sign up" page - this seems empty?

**Comment by tvedebrink** [comment]
Sign in worked fine :+1:

**Comment by tvedebrink** [comment]
When logged in it seems to give access to the admin pages? How and where to register?

**Comment by Amymcdougall** [comment]
I'll hop on my laptop and send you some instructions 😁

**Comment by tvedebrink** [comment]
I think I found it.. On the front page I see links to satuRday events.
Registration of myself i cumbersome - why do I need to type in my own name again? Couldn't there be an "first registrant" tick box and then import my details?

**Comment by tvedebrink** [comment]
The "pay now" is slow.. And only wire transfer possible - credit card should also be available?

**Comment by Amymcdougall** [comment]
Yeah, you have found it! Those are all great ideas that we can look into. Anything you think is a good idea just say and it'll all be added to the tabulation.
